### ISSUE TITLE
What do you think about the inclusion of basic Kanji in future versions?

### ISSUE THREAD

**Issue by aeri** [aeri/Nihonoari-App#7]
I think it might be interesting to include the most basic Kanji such as numbers or the most common ones.

**Comment by seniorm0ment** [comment]
Would be ok with this

**Comment by 90-008** [comment]
I would love this, and could make a PR for it. Would be good to choose exactly what we will include though.

**Comment by aeri** [comment]
@yusdacra Thank you very much for your interest, I had thought of including basic Kanki, like the ones shown [here](https://www.uni-passau.de/fileadmin/dokumente/hsg/nippon/Skripte/kanjibookjlptn5.pdf), and they are the ones needed for the N5 level. 

The question I have is whether given the character of the application the correct answer should be the meaning in English, or accept both Kun'yomi and On'yomi readings. What do you think?

**Comment by 90-008** [comment]
I think it would be better if the user could configure what it accepts. By default it should accept both readings, but if the user wants to practice only Kun`yomi they should be able to choose so, or if the user wants to only practice the English meaning, they should be able to choose so.

**Comment by aeri** [comment]
This could imply a big change in the core of the quiz, with the issue of literal translation still being a problem to link it to a specific language, more so now when I want to touch on the issue of internationalization. #14  It needs to be looked at.

**Comment by aeri** [comment]
I have thought of a way to be able to implement it, once Kanji is selected, through a menu similar to the selection of the set of kanas, select which will be the correct answer to the Kanji. Another doubt I have is whether to allow the coexistence between Kana and Kanji, I would like to make another visual table in which all the Kanji would come out. How do you see it?

**Comment by 90-008** [comment]
I don't think we should allow practicing Kana and Kanji at the same time, since they are both different things. I think it should go like this:
- User opens the app, greeted with a selection between Kana or Kanji
- User selects Kanji
- User now sees Kanji sets: numbers etc. and selects some or all of them, and starts practicing
- User is now at the practice screen which should be like the Kana practice screen, with a visual table button that should show all the Kanji, and the statistics button

I'm a bit busy nowadays so sorry for the late reply :D
